### DISCUSSION TITLE
Release cycle

### DISCUSSION THREAD

**Discussion by dbidwell94** [axios/axios#4719] [General]
What is the general Axios release cycle? I just noticed a feature added to the AxiosStatic instance: The AxiosError! I'm working on mocking axios right now and being able to use the AxiosError class itself is going to be super useful for me. 

I (for now) am using the master branch as a dependency, but when should I expect to see this released so I can go back to using the NPM version? Thanks!

**Comment by jasonsaayman** [comment]
Hi, I am currently trying to get a v1 out, that is what's in the main branch right now. I will think about formalising an actual release cycle after this is out, I think we are pretty close to v1. There should be about a month where this v1 will be in pre-release and testing, then hopefully once released I will be able to finalize a release cyce.
